ui/ozone/public/ozone_platform.h

```diff
@@ -137,6 +139,9 @@ class OZONE_EXPORT OzonePlatform {
       base::Callback<void(const std::vector<gfx::Size>&)>;
   virtual void QueryHostDisplaysData(QueryHostDisplaysDataCallback callback);
 
+  virtual void SetupClipboardDataBridge(
+      ClipboardDataBridge* data, ClipboardDelegate** delegate) {}
```

Comment:
nit: move the definition to ozone_platform.cc as other methods do.

Reply:
done